app/views/budgets/index.html.erb

```diff
@@ -9,7 +9,7 @@
 <% end %>
 
 <% if current_budget.present? %>
-  <div id="budget_heading" class="expanded budget no-margin-top">
+  <div id="budget_heading" class="expanded budget">
```

Comment:
Note removing this class will add an extra margin when there's no banner.

Also, since now we're adding a bottom margin to the banner, IMHO this change isn't necessary :thinking:. What do you think?